storybook/src/Omar/Atoms/Toggle/Toggle.js

```diff
@@ -0,0 +1,58 @@
+import { React, useState } from "react";
```

Comment:
- Add small size to the toggle
- Add label to the toggle 
- Add tests with react-testing-library

Reply:
small size added.
label added.